### PR TITLE
Add timestamp to api timings.

### PR DESCRIPTION
**Pull request by vdbergh**
This was suggested by @vondele.

Also: log api calls, with timestamp, to a file "api.log".

### CODE REVIEW

File: worker/games.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
 import requests
 
 IS_WINDOWS = "windows" in platform.system().lower()
+LOGFILE = os.path.abspath("api.log")
```
Comment (ppigazzini) [inline worker/games.py:28]: If I'm not wrong this can be dropped.

Comment (vdbergh) [reply worker/games.py:28]: Oops.

Comment (vdbergh) [reply worker/games.py]: Fixed.

**Review by ppigazzini — APPROVED** [review]

**Comment by ppigazzini** [comment]
PROD updated to trigger the workers update. Thank you @vdbergh :)

**Comment by vdbergh** [comment]
Thx!

**Pull request merged**